src/pages/account.tsx

```diff
@@ -33,12 +60,8 @@ const IndexPage: React.FC = () => {
           },
         ]}
       />
-      <div>
-        Kimono APP
-        <AmplifySignOut />
-      </div>
     </AmplifyAuthenticator>
   );
 };
 
-export default IndexPage;
+export default AuthStateApp;
```

Comment:
この名前は良いと思うんだけど、Next.jsの `src/page` のComponentはそのままファイル名がそのままURLになるから関数名もそれに合わせた名前が良いかも？🐱

他に合わせる感じだと↓かな？

```suggestion
export default AccountPage;
```

Reply:
ありがとう！なんで`AuthStateApp`にしたんだろう...特にこれにしたかった理由は無いから修正したよ！